modules/io/common/src/main/java/org/locationtech/jts/io/twkb/BoundsExtractor.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2019 Gabriel Roldan, 2022 Aurélien Mino
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * and Eclipse Distribution License v. 1.0 which accompanies this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v20.html
+ * and the Eclipse Distribution License is available at
+ *
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+package org.locationtech.jts.io.twkb;
+
+import org.locationtech.jts.geom.CoordinateSequence;
+import org.locationtech.jts.geom.CoordinateSequenceFilter;
+
+class BoundsExtractor implements CoordinateSequenceFilter {
```

Comment:
I tend to ask for a class javadoc, especially since this is new code.

The (c) 2019 above tends to indicate that this code is coming *from* somewhere, can you tell me where?

Reply:
The code originated from this branch in jts repo: https://github.com/locationtech/jts/tree/twkb
Then it have been reworked/improved by @groldan and pushed as a draft: https://github.com/locationtech/jts/pull/452
I worked from there and polished things, including alignment of implementation on postgis behavior.

The full history commits is here: https://github.com/locationtech/jts/compare/master...murdos:twkb-full-history but I squashed them as per previous discussion https://github.com/locationtech/jts/pull/854#issuecomment-1117935655 to resolve issue with commits of @jnh5y that were using a mail that is not recognized by eclipsefdn/eca bot.

Reply:
Sure I can add javadoc if you want. Note however that this class is not part of the public API, it has a package protected visibility.